kafkacat.c

```diff
@@ -78,20 +79,20 @@ int part_eof_thres = 0;
 
 
 /**
- * Fatal error: print error and exit
+ * Print error and exit if needed
  */
-void RD_NORETURN fatal0 (const char *func, int line,
```

Comment:
We still need FATAL() to point to fatal0() with RD_NORETURN to avoid compilation warnings like this:
https://travis-ci.org/edenhill/kafkacat/jobs/209213737#L1551